docs/installation-docker-compose.md

```diff
@@ -0,0 +1,118 @@
+
+## Docker Compose
+
+The Bindplane Distro for OpenTelemetry (BDOT) Collector can be installed with Docker and Docker Compose.
+
+## Prerequisites
+
+Before installing the Bindplane Distro for OpenTelemetry Collector using Docker Compose, ensure you have the following installed:
+
+- Docker Engine (version 20.10.0 or later)
+- Docker Compose (version 2.0.0 or later)
+
+## Installation Steps
+
+1. Create directories and files to store both Docker Compose and BDOT Collector configuration files:
+
+```
+> config
+    manager.yaml
+> storage
+    config.yaml
+    logging.yaml
+  docker-compose.yaml
+```
+
+Proceed to add content into all files except the `manager.yaml` that will be auto-updated when creating rolling out a config from Bindplane.
+
+2. Paste the following content into your `docker-compose.yaml`:
+
+```yaml
+version: '3.8'
+
+services:
+  bdot-collector:
+    image: ghcr.io/observiq/bindplane-agent:1.76.2
+    command: ["--config=/etc/otel/storage/config.yaml"]
+    volumes:
+      - ./config:/etc/otel/config
+      - ./storage:/etc/otel/storage
+    ports:
+      - "4317:4317"   # OTLP gRPC
+      - "4318:4318"   # OTLP HTTP
+      - "13133:13133" # Health check extension
+      - "55679:55679" # ZPages debugging
+    environment:
+      OPAMP_ENDPOINT: <your-endpoint> # use "wss://app.bindplane.com/v1/opamp" for Bindplane Cloud
+      OPAMP_SECRET_KEY: <your-secret-key>
+      OPAMP_LABELS: install_id=<your-install-id>
+      OPAMP_AGENT_NAME: bdot-collector
+      CONFIG_YAML_PATH: /etc/otel/storage/config.yaml
+      MANAGER_YAML_PATH: /etc/otel/config/manager.yaml
+      LOGGING_YAML_PATH: /etc/otel/storage/logging.yaml
+
+```
+
+Get your keys from the **Agents > Install Agents** page in Bindplane.
+
+![Sample Config](assets/install-keys.png)
+
+3. Paste this into your `config.yaml` file in the `storage` directory:
```

Comment:
Can we clarify that this configuration will be modified by Bindplane and should not be user managed after the initial deployment?